src/style/graphql-query/use-query.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from '@apollo/client';
+import { QueryCategory } from './categories';
+import { GET_CATEGORIES } from './categories';
+
+export const GetCategories = () => {
+  const { loading, error, data } = useQuery<QueryCategory>(GET_CATEGORIES);
+  
+  const categories = data?.categories?.map((category) => ({
+    id: category.id,
+    imageUrl: category.imageUrl,
+    name: category.name,
+    numberOfProperties: category.numberOfProperties,
+  }));
```

Comment:
como vc ta mapeando para o mesmo objeto, poderia fazer simplesmente isso:
```suggestion
  const categories = data?.categories ?? [];
```